jenkins/ci.opensuse.org/openstack-devstack.yaml

```diff
@@ -53,6 +53,17 @@
               du $file
               exit 1
           fi
+          scp root@cleanvm:/opt/stack/results.html .
+          scp root@cleanvm:/opt/stack/results.xml .
```

Comment:
Maybe use a special directory like .artifacts instead of cluttering up the workspace

Reply:
I used to store the workspaces because they were cleaned automatically, are we wiping it?

Reply:
I'm not 100% sure the workspace is wiped, it might depend on the job